<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputDocuments</class>
 <widget class="QDialog" name="OutputDocuments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вихідні документи</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Історія платежів</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QListWidget" name="listPaymentsHistory"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pbPaymentsHistory">
         <property name="text">
          <string>Зберегти в файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Список боржників</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QListWidget" name="listDebtors"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pbDebtors">
         <property name="text">
          <string>Зберегти в файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Сортування</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Вивести всіх клієнтів з міста:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="cbSities">
         <item>
          <property name="text">
           <string>Маріуполь</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Покровськ</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Авдіївка</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QListWidget" name="listSort"/>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QPushButton" name="pbSort">
         <property name="text">
          <string>Зберегти в файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Прайс-лист</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Вартість підключення (багатоповерхівки):</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spCities">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>2000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Вартість підключення (приватний сектор):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spPrivateSector">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>2000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Виклик фахівця на об'єкт:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spMaster">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>2000</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Метр звитої пари:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spEthernet">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>2000</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Налаштування ОС:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spOperatingSystem">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>2000</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Налаштування обладнання:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="spHardware">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>2000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pbPrice">
         <property name="text">
          <string>Зберегти в файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pbUpdate">
     <property name="text">
      <string>Оновити</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
